airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FormattedMessage } from "react-intl";
+
+import { Button } from "components/ui/Button";
+import { Text } from "components/ui/Text";
+
+import { SourceDefinitionRead } from "core/request/AirbyteClient";
+import { useDocumentationPanelContext } from "views/Connector/ConnectorDocumentationLayout/DocumentationPanelContext";
+
+import styles from "./ResourceNotAvailable.module.scss";
+import octaviaWorker from "../octavia-worker.png";
+import { useAnalyticsTrackFunctions } from "../useAnalyticsTrackFunctions";
+
+export const ResourceNotAvailable: React.FC<
+  React.PropsWithChildren<{
+    activeTab: "erd" | "schema";
+    isRequested: boolean;
+    setRequested: (val: boolean) => void;
+  }>
+> = ({ activeTab, setRequested, isRequested }) => {
+  const { selectedConnectorDefinition } = useDocumentationPanelContext();
+  const { trackRequest } = useAnalyticsTrackFunctions();
+
+  return (
+    <div className={styles.requestContainer}>
+      <img src={octaviaWorker} alt="" />
+      {isRequested ? (
+        <Text size="lg">
+          <FormattedMessage id="sources.request.thankYou" />
+        </Text>
+      ) : (
+        <>
+          <Text size="lg">
+            <FormattedMessage id="sources.request.prioritize" />
+          </Text>
+          <FontAwesomeIcon icon={faArrowDown} />
```

Comment:
Design nitpick: I think we should just remove this icon. I don't think we have something with this arrow leading to the button anywhere else, and I feel even without this looks how we'd want it:

![screenshot-20230213-134507](https://user-images.githubusercontent.com/877229/218461048-79f04f5c-bc76-485e-9947-653541cb4ed8.png)